src/components/crag/cragSummary.tsx

```diff
@@ -298,8 +314,23 @@ export default function CragSummary (props: AreaType): JSX.Element {
             formAction={FormAction}
           />
 
-          <div className='mt-4 text-right' id='editTogglePlaceholder'>
-            <EditModeToggle onChange={setEditMode} />
+          <div className='mt-4 justify-end flex flex-row' id='editTogglePlaceholder'>
+            <div className='mr-2' id='editTogglePlaceholder'>
+              <EditModeToggle onChange={setEditMode} />
+            </div>
+            {!isEmpty(changeHistory) && (
+              <MobileDialog>
+                <DialogTrigger>
+                  <Button
```

Comment:
Can you please use `<button className='btn btn-secondary ...'>`?  
See https://daisyui.com/components/button/

We're deprecating the custom Button component.  Sorry this is not documented anywhere.

Reply:
@vnugent Thanks for hte feedback! 
I've updated the PR with the button you suggested and removed the dynamic data fetching.